### PR TITLE
Scan for handoff

### PR DESCRIPTION
**Pull request by naviza**
https://drive.google.com/file/d/1X28vmZL2yt8WTBnc3fepqqqYNvxzWOlu/view
Video of basic handoff

### CODE REVIEW

File: app/src/main/java/com/example/bookwormadventuresdeluxe2/RequestDetailViewFragment.java [hunk 1]
```diff
@@ -41,17 +44,21 @@
     private Button btn2;
     private TextView exchange;
     private DocumentReference bookDocument;
+    private boolean goodScan;
```
Comment (JawadHossain) [inline app/src/main/java/com/example/bookwormadventuresdeluxe2/RequestDetailViewFragment.java:47]: exchange, goodScan, resouces are not used. Can u remove them pls?

File: app/src/main/java/com/example/bookwormadventuresdeluxe2/RequestDetailViewFragment.java [hunk 2]
```diff
@@ -41,17 +44,21 @@
...
     private RequestDetailViewFragment requestDetailViewFragment;
     private Resources resources;
     private ConstraintLayout dropdownContainer;
 
+    public static final int REQUEST_GIVE_SCAN = 5;
+    public static final int REQUEST_RECIEVE_SCAN = 6;
```
Comment (JawadHossain) [inline app/src/main/java/com/example/bookwormadventuresdeluxe2/RequestDetailViewFragment.java:53]: Pls correct typo to REQUEST_RECEIVE_SCAN. Select this, press and hold ALT J to select all occurrences and change at once.

File: app/src/main/java/com/example/bookwormadventuresdeluxe2/RequestDetailViewFragment.java (hunk 1 above)
Comment (naviza) [reply app/src/main/java/com/example/bookwormadventuresdeluxe2/RequestDetailViewFragment.java:47]: I get this too... goodScan I can delete but exchange and resources are both used (line 65 and 78)

Comment (JawadHossain) [reply app/src/main/java/com/example/bookwormadventuresdeluxe2/RequestDetailViewFragment.java:47]: ok thanks!

File: app/src/main/java/com/example/bookwormadventuresdeluxe2/RequestDetailViewFragment.java [hunk 3]
```diff
@@ -41,17 +44,21 @@
...
     private static int SetLocationActivityResultCode = 7;
 
     public RequestDetailViewFragment()
     {
         // Required empty public constructor
     }
 
+
```
Comment (JawadHossain) [inline app/src/main/java/com/example/bookwormadventuresdeluxe2/RequestDetailViewFragment.java:61]: pls remove blank line

File: app/src/main/java/com/example/bookwormadventuresdeluxe2/RequestDetailViewFragment.java (hunk 2 above)
Comment (naviza) [reply app/src/main/java/com/example/bookwormadventuresdeluxe2/RequestDetailViewFragment.java:53]: ngl that's kind of embarrassing for me but thanks for catching this

**Review by JawadHossain — APPROVED** [review]

**Review by huddy987 — APPROVED** [review]

**Pull request merged**